tests/generated_results/inplace_risk_test.xml

```diff
@@ -187,4 +187,4 @@
     <ns0:score maximum="100.000000" system="urn:xccdf:scoring:default">49.081196</ns0:score>
     <ns0:score maximum="70.000000" system="urn:xccdf:scoring:flat">40.000000</ns0:score>
   </ns0:TestResult>
-</ns0:Benchmark>
+</ns0:Benchmark>
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/commit/194ab3ecbeb1ff37d2787648f732f830eda3b1b5